akka-docs/src/main/paradox/io.md

```diff
@@ -92,28 +92,28 @@ not error handling. In other words, data may still be lost, even if every write
 
 ### ByteString
 
-To maintain isolation, actors should communicate with immutable objects only. `ByteString` is an
+To maintain isolation, actors should communicate with immutable objects only. @apidoc[ByteString](akka.util.ByteString) is an
```

Comment:
I think it would be better to just link the first occurrence here, and leave the rest as code snippets. Not super useful with 50 links in the same section to the same API-doc IMO.

Reply:
Let's try to do it :) . It should be better.

Reply:
Much nicer!